internal/core/command/v2/mockclients/interfaces/mocks/MockDeviceClient.go

```diff
@@ -0,0 +1,210 @@
+// Code generated by mockery v2.5.1. DO NOT EDIT.
+
+package mocks
```

Comment:
These mocks should be in core-contracts so they can be used through out edgex.

Reply:
Agree.  I've submitted a PR at https://github.com/edgexfoundry/go-mod-core-contracts/pull/457 to create mocks at go-mod-core-contracts.